### PR TITLE
[pota-quantization-value-test] Introduce qconfig test

### PR DESCRIPTION
**Pull request by jinevening**
This introduces mixed-precision qconfig tests.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/8596
Draft PR: #8656

### CODE REVIEW

File: compiler/pota-quantization-value-test/test_fake_wquant_with_config.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  rm -f "${PASSED_TAG}"
+
+  cat > "${TEST_RESULT_FILE}_fake_quantization_with_config.log" <(
+    exec 2>&1
+    set -ex
+
+    # Run circle-quantizer with --quantize_dequantize_weights
+    "${CIRCLE_QUANTIZER_PATH}" \
+      --quantize_dequantize_weights float32 "${DTYPE}" "${GRANULARITY}" \
+      --config "${SOURCE_PATH}/config_files/${MODELNAME}/${GRANULARITY}/${DTYPE}/qconf.json" \
```
Comment (jinevening) [inline compiler/pota-quantization-value-test/test_fake_wquant_with_config.sh:49]: This is a major difference from `test_fake_wquant.sh`. `circle-quantizer` is invoked with `--config` option.

File: compiler/pota-quantization-value-test/test_fake_wquant_with_config.sh [hunk 2]
```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# This script tests fake quantization with config file
+#
+# HOW TO USE
+#
+# ./test_fake_quantization.sh <path/to/test.config> <path/to/work_dir> <TEST 1> <TEST 2> ...
```
Comment (seanshpark) [inline compiler/pota-quantization-value-test/test_fake_wquant_with_config.sh:7]: maybe `./test_fake_wquant_with_config.sh ...` ?

File: compiler/pota-quantization-value-test/test_quantization_with_config.sh [hunk 3]
```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# This script tests quantize_with_minmax option of circle-quantizer with config file
+#
+# HOW TO USE
+#
+# ./test_quantization.sh <path/to/test.config> <path/to/work_dir> <TEST 1> <TEST 2> ...
```
Comment (seanshpark) [inline compiler/pota-quantization-value-test/test_quantization_with_config.sh:7]: here too

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**